app/models/attraction.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+#
+# Attraction is the superclass for all (touristic) attractions in the smart village.
+#
+class Attraction < ApplicationRecord
+  belongs_to :category
+  has_many :adresses, as: :adressable
+  has_many :contacts, as: :contactable
+  has_many :media_contents, as: :mediaable
+  has_one :operating_company, as: :companyable
+  has_one :data_provider, as: :provideable
+  has_many :web_url, as: :web_urlable
+
+  validates_presence_of :name, :active
```

Comment:
hatten wir nicht gesagt wir wollten `active` aus der Validation raus nehmen und in der Migration `:active, default: true`setzen?

Reply:
Ja

Reply:
default value existiert schon, muss also nur hier raus